.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "3.5.2"
+# command to install dependencies
+install: 
+  - pip install -r requirements.txt
+  - pip install coveralls
+# command to run tests
+script: 
+  - nosetests --with-coverage --cover-erase --cover-html --cover-package=app
```

Comment:
What do the --cover-erase and --cover-html do?
I feel --cover-package should be enough.

Also consider using [*tox*](https://tox.readthedocs.io/en/latest/) to automate these tests.